Adyen/Utilities/Analytics.swift

```diff
@@ -25,10 +25,10 @@ public class Analytics {
         public var flavor: Flavor
 
         /// :nodoc:
-        public var environment: Environment
+        public var environment: AnyAPIEnvironment
```

Comment:
We can use `ApiContext` here or at least we can add overload for 'init' with `ApiContext`
It is more common pattern.

Reply:
Good point. I think `Analytics` can be refactored a little bit as well.